E2E_Tests/pageObjects/addToDoListItemPage.ts

```diff
@@ -0,0 +1,66 @@
+import { expect, type Locator, type Page } from '@playwright/test'
+
+export default class AddToDoListItemPage {
+  private page: Page
+
+  readonly pageHeader: Locator
+  readonly enterTitle: Locator;
+  readonly enterNotes: Locator;
+  readonly enterDay: Locator;
+  readonly enterMonth: Locator;
+  readonly enterYear: Locator;
+  readonly addTaskButton: Locator;
+  readonly warningHeader: Locator;
+  readonly warningTitle: Locator;
+  readonly warningTitleError: Locator;
+  readonly dueDateError: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.pageHeader = page.locator('h1', { hasText: 'Add a to-do list item' });
+    this.enterTitle = page.locator('#title');
+    this.enterNotes = page.locator('#notes');
+    this.enterDay = page.locator('#due-date-day');
+    this.enterMonth = page.locator('#due-date-month');
+    this.enterYear = page.locator('#due-date-year');
+    this.addTaskButton = page.locator('#submit-button');
+    this.warningHeader = page.locator('#main-content > div > div > div > div > h2');
```

Comment:
these look a bit fragile, want me to help you add some test selectors?

Reply:
Let's do it yeah!